formats/frontend_links_definition.json

```diff
@@ -47,11 +60,7 @@
       "links": {
         "type": "object",
         "additionalProperties": true,
-        "properties": {
-          "parent": {
-            "$ref": "#/definitions/guid_list"
-          }
-        }
+        "description": "TODO: investigate if could this be validated recursively."
```

Comment:
I'm fairly sure the answer to this is no, definitely with current tooling. Probably within the scope of JSON schemas themselves :(